app/views/docs/getting-started-for-server.phtml

```diff
@@ -41,9 +57,24 @@
     </li>
     <li>
         <h3>Dart</h3>
+        <div class="ide margin-top-small" data-lang="bash" data-lang-label="Bash">
+            <pre class="line-numbers"><code class="prism language-bash" data-prism>dart pub add dart_appwrite</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Kotlin</h3>
         <div>
-            <div class="ide margin-top-small" data-lang="dart" data-lang-label="Dart SDK">
-                <pre class="line-numbers"><code class="prism language-dart" data-prism>dart pub add dart_appwrite</code></pre>
+            <p>Add maven repositories to your `build.gradle(.kts)` file</p>
+            <div class="ide margin-top-small" data-lang="gradle" data-lang-label="Gradle">
+                <pre class="line-numbers"><code class="prism language-gradle" data-prism>repositories {
+    mavenCentral()
+}</code></pre>
```

Comment:
We need to also add the repository for snapshot releases. 
Please refer to the readme on sdk-for-kotlin ( dev branch )

Reply:
Do we?

Reply:
Okay. Thinking about it again, it doesn't need to be on the Getting started guide :+1: